### DISCUSSION TITLE
Error in unserialize(socklist[[n]]) : error reading from connection Calls: %dopar% ... recvOneData -> recvOneData.SOCKcluster -> unserialize Execution halted

### DISCUSSION THREAD

**Discussion by whitneyt1** [angelolab/ark-analysis#769] [Q&A] [answered]
Hello! 

I updated ark, received this error when "using pixel SOM weights to assign pixel clusters", redownloaded ark from scratch and still received this error. I have 11 400x400 FOVs across 5 run folders, which have been processed through toffy then run through the new ark package from the first segmentation notebook. 

<img width="988" alt="Screen Shot 2022-10-13 at 11 54 30 AM" src="https://user-images.githubusercontent.com/46573884/195682468-2460dec8-c0d8-4409-8aaa-f77e9f5a3080.png">

Please let me know what other information would be helpful to provide. Thank you!

**Comment by ngreenwald** [comment, marked as answer]
Hey @whitneyt1, this is a problem with the parallel processing library we're using to speed up these calculations. This has been a huge headache to maintain, so we're going to get rid of it, see #771 for progress updates.

**Comment by whitneyt1** [comment]
Thank you!!!

**Reply by alex-l-kong** [reply]
@whitneyt1 we recently pushed a change that defaults the processing to single core. There are optional arguments that you can set to turn multiprocessing back on if desired.
